acceptance/acceptance_test.go

```diff
@@ -454,19 +449,43 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 
 	printedRepls := false
 
+	// Compute repls for UUIDs. For each unique UUID we compute a unique replacement.
+	uuidRegex := regexp.MustCompile(`[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}`)
```

Comment:
It seems like these repls won't be recorded ion repls.json file which we do above, is it intended?